spec/factories/user_tickets.rb

```diff
@@ -1,5 +1,8 @@
 FactoryGirl.define do
   factory :user_ticket do
     ticket_number "MyString"
+    event_source "Facebook"
+    booking_id 1
+    ticket_type_id 1
```

Comment:
Look into using Faker for getting random attributes here. I can see that it's already included into the project.